client/src/index.jsx

```diff
@@ -33,9 +33,10 @@ function App() {
   const [flights, setFlights] = useState([]);
   const [hotels, setHotels] = useState([]);
   const classes = useStyles();
+  const [favorites, setFavorites] = useState([])
 
   useEffect(() => {
-    axios.get('/userdata')
+    axios.get('/user/data')
       .then((res) => {
         if (res.data === '') {
           setUser('');
```

Comment:
setUser sounds like your storing user information but you are only storing the name